Content.Shared/Follower/FollowerSystem.cs

```diff
@@ -249,20 +250,36 @@ public void StopFollowingEntity(EntityUid uid, EntityUid target, FollowedCompone
     }
 
     /// <summary>
-    /// Get the most followed entity.
+    /// Gets the entity with the most non-admin ghosts following it.
     /// </summary>
-    public EntityUid? GetMostFollowed()
+    public EntityUid? GetMostGhostFollowed()
     {
+        // Keep a tally of how many ghosts are following each entity
+        var followedEnts = new Dictionary<EntityUid, int>();
+
+        // Look for followers that are ghosts and are player controlled
+        var query = EntityQueryEnumerator<FollowerComponent, GhostComponent, ActorComponent>();
+        while (query.MoveNext(out var _, out var follower, out var ghost, out var _))
+        {
+            // Exclude admin ghosts
+            if (!ghost.CanGhostInteract)
+            {
+                var followed = follower.Following;
+                // Add new entry or increment existing
+                if (!followedEnts.TryAdd(followed, 1))
+                    followedEnts[followed]++;
```

Comment:
Is this safe?

Reply:
Ehh, possibly not. I think I found a better way to do an upsert though.